browser/ui/views/split_view/split_view_browsertest.cc

```diff
@@ -226,13 +225,27 @@ IN_PROC_BROWSER_TEST_F(SideBySideEnabledBrowserTest, SelectTabTest) {
   EXPECT_TRUE(tab_strip()->tab_at(3)->IsActive());
   EXPECT_TRUE(split_view_separator()->GetVisible());
 
-  // Chromium's mini toolbar should be hidden always as we're using own own mini
-  // urlbar.
-  EXPECT_FALSE(
+  // Chromium's mini toolbar should be visible.
+  EXPECT_TRUE(
       brave_multi_contents_view()->mini_toolbar_for_testing(0)->GetVisible());
-  EXPECT_FALSE(
+  EXPECT_TRUE(
       brave_multi_contents_view()->mini_toolbar_for_testing(1)->GetVisible());
 
+  // Check mini toolbar uses our menu model.
+  brave_multi_contents_view()->mini_toolbar_for_testing(0)->OpenSplitViewMenu();
+  auto* menu_model =
+      static_cast<BraveSplitTabMenuModel*>(brave_multi_contents_view()
+                                               ->mini_toolbar_for_testing(0)
+                                               ->menu_model_.get());
```

Comment:
If something goes wrong and `SplitTabmenuModel` is used instead of `BraveSplitTabMenuModel`, isn't it bad to cast the menu model to `BraveSplitTabMenuModel` anyway? I guess we can do without casting here?